### PR TITLE
Add config toggle.

### PR DESCRIPTION
**Pull request by qarlosalberto**
Issue: https://github.com/particle-iot/console-panel/issues/10

### CODE REVIEW

File: lib/console-manager.js [hunk 1]
```diff
@@ -16,6 +16,7 @@ export default class ConsoleManager {
 
 	// Toggle console panel
 	toggle() {
+		alert("hola")
```
Comment (suda) [inline lib/console-manager.js]: I think this was for debug  purposes and is no longer needed 😅

File: lib/config/config-schema.json [hunk 2]
```diff
@@ -0,0 +1,7 @@
+{
+  "Show": {
```
Comment (suda) [inline lib/config/config-schema.json]: Could you make the key lowercase? Atom's pattern [seems to be camel case with first letter lowercase](https://atom.io/docs/api/v1.6.0/Config)

File: lib/console-manager.js (hunk 1 above)
Comment (qarlosalberto) [reply lib/console-manager.js]: looool! Yes :/

Comment (qarlosalberto) [reply lib/console-manager.js]: fixed!

File: lib/config/config-schema.json (hunk 2 above)
Comment (qarlosalberto) [reply lib/config/config-schema.json]: Fixed!

**Comment by suda** [comment]
Thanks Carlos!

**Pull request merged**